server/static/dev.html

```diff
@@ -37,6 +37,12 @@
         window.document.querySelector('#root').style.background = "#111319";
       }
     </script>
+    <script type="module">
+      // PWA: https://www.pwabuilder.com/serviceworker
+      import 'https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate';
+      const el = document.createElement('pwa-update');
+      document.body.appendChild(el);
+    </script>
```

Comment:
The UI that this provides doesn't match the rest of the application, I think we'd need to find a way to have it use the regular toast system that is already built in.

I feel like the knock on affects of potentially serving cached assets and js bundles might be quite significant and unpredictable.

Reply:
Sure, this is not very unrelated to the above right? If you can guide me to a tutorial how to do this, I'll implement it. Thanks.